server.js

```diff
@@ -1,10 +1,8 @@
-var express = require('express')
-var app = express()
+var express = require('express');
+var app = express();
 
 app.get('/', function (req, res) {
```

Comment:
(Error) Line 6: Expected linebreaks to be 'LF' but found 'CRLF'. - linebreak-style